### PR TITLE
update the translation of `Object.keys()`

### PR DESCRIPTION
**Pull request by robert-jx**
### Description

update the translation of `Object.keys()`
### Motivation

update the translation of `Object.keys()`
### Additional details

none

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/object/keys/index.md [hunk 1]
```diff
@@ -5,70 +5,79 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/keys
...
+// 字符串具有索引作为可枚举的自身属性
+console.log(Object.keys("foo")); // ['0', '1', '2']
 
-// In ES2015+
-Object.keys('foo');  // ["0", "1", "2"]
+// 其他基本类型没有自己的属性
+console.log(Object.keys(100)); // []
 ```
 
+> **Note:** 在 ES5 中，将一个非对象传递给 `Object.keys()` 会抛出一个 {{jsxref("TypeError")}}。
```
Comment (skyclouds2001) [inline files/zh-cn/web/javascript/reference/global_objects/object/keys/index.md:79]:
```suggestion
> **注意：** 在 ES5 中，将一个非对象传递给 `Object.keys()` 会抛出一个 {{jsxref("TypeError")}}。
```

**Comment by robert-jx** [comment]
@JasonLamv-t

File: files/zh-cn/web/javascript/reference/global_objects/object/keys/index.md [hunk 2]
```diff
@@ -5,70 +5,79 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/keys
...
-如果你想获取一个对象的所有属性，甚至包括不可枚举的，请查看 {{jsxref("Object.getOwnPropertyNames")}}。
+如果你想要 _所有_ 字符串键自身的属性，包括不可枚举的属性，参见 {{jsxref("Object.getOwnPropertyNames()")}}。
 
-## 注意
+### 在基本类型中使用 Object.keys()
 
-在 ES5 里，如果此方法的参数不是对象（而是一个原始值），那么它会抛出 {{jsxref("TypeError")}}。
-
-在 ES2015 中，非对象的参数将被强制转换为一个对象。
+非对象参数会[强制为对象](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object#强制对象)。只有字符串可以有自己的可枚举属性，而所有其他基本类型都返回一个空数组。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/keys/index.md:69]:
```suggestion
非对象参数会[强制转换为对象](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object#对象强制转换)。只有字符串可以有自己的可枚举属性，而其他所有基本类型都返回一个空数组。
```

Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/keys/index.md:69]:
```suggestion
非对象参数会[强制转换为对象](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object#对象强制转换)。只有字符串可以有自己的可枚举属性，而其他所有基本类型都返回一个空数组。
```

File: files/zh-cn/web/javascript/reference/global_objects/object/keys/index.md [hunk 3]
```diff
@@ -5,70 +5,79 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/keys
...
 
 ```js
-// In ES5
-Object.keys('foo');  // TypeError: "foo" is not an object
+// 字符串具有索引作为可枚举的自身属性
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/keys/index.md:72]:
```suggestion
// 字符串具有索引作为可枚举的自有属性
```

File: files/zh-cn/web/javascript/reference/global_objects/object/keys/index.md [hunk 4]
```diff
@@ -5,70 +5,79 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/keys
...
+console.log(Object.keys("foo")); // ['0', '1', '2']
 
-// In ES2015+
-Object.keys('foo');  // ["0", "1", "2"]
+// 其他基本类型没有自己的属性
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/keys/index.md:75]:
```suggestion
// 其他基本类型没有自有属性
```

**Review by JasonLamv-t — COMMENTED** [review]
Some suggestions.

**Review by yin1999 — APPROVED** [review]
Thanks. Sorry for the delay ;)

**Comment by robert-jx** [comment]
> Thanks. Sorry for the delay ;)

Never mind😁

**Pull request merged**